### PR TITLE
fix Scheduler: T1097917

### PR DESCRIPTION
**Pull request by GoodDayForSurf**

### CODE REVIEW

File: packages/devextreme-react/src/core/dx-template.ts [hunk 1]
```diff
@@ -35,6 +37,23 @@ function createDxTemplate(
 
       let templateId: string;
 
+      const onRemoved = (): void => {
```
Comment (dxvladislavvolkov) [inline packages/devextreme-react/src/core/dx-template.ts:40]: Looks like we need add a test to cover this code.

File: packages/devextreme-react/src/core/dx-template.ts [hunk 2]
```diff
@@ -56,11 +79,9 @@ function createDxTemplate(
           {
             content: contentProvider(props),
             container,
-            onRemoved: () => {
-              templatesStore.setDeferredRemove(templateId, true);
-              renderedTemplates.delete({ key1: data.model, key2: container });
-            },
+            onRemoved: onRemoved,
```
Comment (dxvladislavvolkov) [inline packages/devextreme-react/src/core/dx-template.ts:82]: Use shorter syntaxis. Just { onRemoved }

**Review by dxvladislavvolkov — CHANGES_REQUESTED** [review]

**Review by ivanblinov2k17 — APPROVED** [review]

**Review by GoodDayForSurf — COMMENTED** [review]
поменял

**Review by GoodDayForSurf — COMMENTED** [review]
добавил

**Review by dxvladislavvolkov — APPROVED** [review]

**Pull request merged**